src/MVVM/Factories/ViewModelFactory.cs

```diff
@@ -121,19 +204,62 @@ internal static INavigationService TryResolveNavigationServiceInstance()
         }
 
 
+
+        /// <summary>
+        /// Creates a new ViewModel instance and invokes the <see cref="IViewModel{TModel}.InitializeAsync(TModel)"/> method.
+        /// </summary>
+        /// <typeparam name="TViewModel">Type of the ViewModel to create</typeparam>
+        /// <typeparam name="TModel">Type of the parameter that will be passed in the <see cref="IViewModel{TModel}.InitializeAsync(TModel)"/> method</typeparam>
```

Comment:
Codacy found an issue: [Split this 161 characters long line (which is greater than 130 authorized).](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5767060)